ci/testnet-automation.sh

```diff
@@ -28,7 +29,11 @@ launchTestnet() {
   net/init-metrics.sh -e
 
   echo --- start "$nodeCount" node test
-  net/net.sh start -o noValidatorSanity -S solana_*.snap
+  if [[ -n "$PREBUILT" ]]; then
```

Comment:
where does PREBUILT get set?

Reply:
It's pipeline env variable. I added a new pipeline (solana-testnet-prebuilt)